client/state/jetpack/credentials/test/reducer.js

```diff
@@ -0,0 +1,46 @@
+import { markCredentialsAsInvalid, markCredentialsAsValid, testCredentials } from '../actions';
+import { testRequestStatus } from '../reducer';
+
+describe( 'reducer', () => {
+	describe( 'testRequestStatus', () => {
+		test( 'should return the initial state', () => {
+			expect( testRequestStatus( undefined, {} ) ).toEqual( {} );
+		} );
+
+		test( 'state should be valid if the credentials are okay', () => {
+			const stateIn = undefined;
```

Comment:
How can `stateIn` be `undefined` when the default state of the reducer is an empty object?

Reply:
You are right! Fixing this.

Reply:
Fixed!